### ISSUE TITLE
Unavailable RAR file password for the data

### ISSUE THREAD

**Issue by MMuzammilAzad** [ShyTHU/GFresNet#1]
Its mentioned that the RAR file password is mentioned at the end of the article's abstract. However, upon viewing the paper, I couldn't find the password for the RAR file. Can you please mention the password to open the RAR file for the data.

Thank you

**Comment by abdulsami1951** [comment]
Yes, Same issue with me. I also cannot find the password of the RAR file. Kindly guide me for the password??
